src/develop/masks/masks.c

```diff
@@ -2137,6 +2147,17 @@ static void _menu_add_exist(dt_iop_module_t *module, int formid)
   dt_masks_set_edit_mode(module, DT_MASKS_EDIT_FULL);
 }
 
+void dt_masks_group_update_name(dt_iop_module_t *module)
+{
+  dt_masks_form_t *grp = dt_masks_group_from_module(module);
+  if (!grp)
+    return;
+
+  _dt_masks_group_set_name_from_module(module, grp);
+  dt_dev_add_masks_history_item(darktable.develop, module, TRUE);
+  dt_masks_iop_update(module);
```

Comment:
I'm not completely sure whether this is correct after renaming a form/group.
Do I need to also call `_check_id(grp);` like in the function which creates a new group (`dt_masks_group_create()`)?

Reply:
I don't think so, but hard to tell as there is missing comments on _check_id() but it seems only to ensure that we have no duplicate id. Since here you just change the name it seems ok.

Reply:
I agree, that's what my thought was too :+1: